ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -57,4 +57,13 @@ QtObject {
         }
         return hovered? WalletUtils.colorizedChainPrefix(chainShortNames) + Utils.richColorText(finalAddress, Theme.palette.directColor1) : chainShortNames + finalAddress
     }
+
+    // compensate for the amount of eth to pay for the gas
+    function prepareForMaxSend(value, symbol) {
```

Comment:
maybe `calculateMaxSafeSendAmount`? `prepareForMaxSend` is quite vague, especially outside context of max button.
I would also add explanation that `value` is maximum value intended to use for the transaction, usually total balance for given token.

Reply:
Alright